bitrise/setup.go

```diff
@@ -40,23 +42,21 @@ var PluginDependencyMap = map[string]PluginDependency{
 	},
 }
 
-// RunSetupIfNeeded ...
-func RunSetupIfNeeded(appVersion string, isFullSetupMode bool) error {
```

Comment:
`isFullSetupMode` is always false, so I removed it

Reply:
also `appVersion` seems to be unused